pkgs/tools/admin/trivy/default.nix

```diff
@@ -20,16 +18,13 @@ buildGoModule rec {
 
   excludedPackages = "misc";
 
-  buildInputs = lib.optionals (stdenv.isDarwin && stdenv.isx86_64)
-    [ CoreFoundation Security ];
```

Comment:
Please leave this in. It is very likely that we still need it.

Reply:
This was my attempted fix that did nothing and was merged too early

This isnt a rust project :)